akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala

```diff
@@ -271,7 +350,7 @@ final class ORSet[A] private[akka] (
    * INTERNAL API
    */
   private[akka] def clear(node: UniqueAddress): ORSet[A] =
-    assignAncestor(copy(elementsMap = Map.empty))
+    assignAncestor(copy(elementsMap = Map.empty)) // FIXME delta?
```

Comment:
yes, I'll give it a try tomorrow morning, but I think it can be worth representing the clear operation as a separate delta type. It should be enough to send the `vvector` and do a full state merge on the other side (it's like sending full state).